acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202301.2
+RUN pip uninstall -y onnxruntime-training
+RUN pip install -i https://test.pypi.org/simple/ onnxruntime-training==1.14.0.dev20230119001
+
+# ENV NCCL_DEBUG=INFO
+ 
+# ls root dir
+RUN ls -lh
+
+USER root
+RUN apt-get -y update
+
+# pip install requirements
+RUN pip install https://automlcesdkdataresources.blob.core.windows.net/finetuning-models/prp_wheels/azureml_acft_accelerator-0.1.0.0-py3-none-any.whl
```

Comment:
Please get the azureml_acft_accelerator and onnxruntime-training packages into regular PyPI.

References to wheels in storage blobs shouldn't be in azureml-assets.

Reply:
+1